ios/Classes/PdftronFlutterPlugin.h

```diff
@@ -171,6 +177,25 @@ static NSString * const PTAnnotationFlagPrintKey = @"print";
 static NSString * const PTAnnotationFlagReadOnlyKey = @"readOnly";
 static NSString * const PTAnnotationFlagToggleNoViewKey = @"toggleNoView";
 
+// Default annotation toolbar names.
+typedef NSString * PTDefaultAnnotationToolbarKey;
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarView = @"PDFTron_View";
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarAnnotate = @"PDFTron_Annotate";
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarDraw = @"PDFTron_Draw";
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarInsert = @"PDFTron_Insert";
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarFillAndSign = @"PDFTron_Fill_and_Sign";
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarPrepareForm = @"PDFTron_Prepare_Form";
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarMeasure = @"PDFTron_Measure";
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarPens = @"PDFTron_Pens";
+static const PTDefaultAnnotationToolbarKey PTAnnotationToolbarFavorite = @"PDFTron_Favorite";
```

Comment:
All of the actual strings in this file seem to follow different conventions. The strings themselves only need to be unique within this file, so a "PDFTron_" doesn't add anything useful. I would suggest making the string something like "toolbarGroupView" so that it wouldn't accidental'y collide with another key in this file that had something to do with "View", like a button that switched to view or something.

Reply:
If for some reason it helps that these are the same as the Dart strings, then you can leave these as-is.